.gitignore

```diff
@@ -15,3 +15,6 @@
 /log/*
 !/log/.keep
 /tmp
+
+# Ignore Simplecov coverage report
```

Comment:
I wanted to have the comment so it would match the rest of the .gitignore file. I think it looks better this way.